adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AnySAMOutFormatter.scala

```diff
@@ -29,7 +29,7 @@ import scala.collection.mutable.ListBuffer
  * An OutFormatter that automatically infers whether the piped input is SAM or
  * BAM. Autodetecting streamed CRAM is not currently supported.
  */
-class AnySAMOutFormatter extends OutFormatter[AlignmentRecord] {
+case class AnySAMOutFormatter(stringency: ValidationStringency = ValidationStringency.STRICT) extends OutFormatter[AlignmentRecord] {
```

Comment:
From the Jenkins failures, looks like the default ctr parameter value isn't available to Python.  Could you add a no-arg `apply` method or update the Python code?